README.md

```diff
@@ -1,9 +1,6 @@
 # Northcoders News API
 
 For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
-
-
-
---- 
-
 This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+
+1. Once you have cloned the repo, please add your own copy of .env.development and .env.test in the root folder of the project in order to link correctly the databases. Use the following PGDATABASE=data_base_name and PGDATABASE=data_base_name_test respectively.
```

Comment:
Dont forget to include how to install all dependencies listed in your package.json using the command `npm install` as well